client/.env

```diff
@@ -1 +1,4 @@
-NODE_PATH=./src
+DB_USERNAME=root
```

Comment:
@Marifer-Enriquez I think we shouldn't be performing changes to this file. The one for setting up the env variables should be in the root folder.